docs/source/old_style.rst

```diff
@@ -0,0 +1,127 @@
+Old Style
```

Comment:
yes this is the style that refactordoc uses, but I want to discourage its use. I could call it something  like `simple`.